10.9-libcxx/stable/main/finkinfo/libs/pythonmods/packaging-py.info

```diff
@@ -0,0 +1,52 @@
+Info3: <<
+
+Package: packaging-py%type_pkg[python]
+Version: 17.1
+
+Revision: 1
+Homepage: https://github.com/pypa/packaging
+Maintainer: Derek Homeier <dhomeie@gwdg.de>
+Type: python (2.7 3.4 3.5 3.6)
+Depends: python%type_pkg[python], pyparsing-py%type_pkg[python], six-py%type_pkg[python]
+BuildDepends: setuptools-tng-py%type_pkg[python]
+
+Source: https://pypi.io/packages/source/p/packaging/packaging-%v.tar.gz
+Source-MD5: 8baf8241d1b6b0a5fae9b00f359976a8
+
+Description: Core utilities for Python packages
+DescDetail: <<
+This module includes APIs for
+-  Version handling (`.version.Version`)
+-  Specifiers (`.specifiers.SpecifierSet`)
+-  Markers (`.markers.Marker`)
+-  Requirements (`.requirements.Requirement`)
+-  Utilities (`.utils`)
+<<
+
+CompileScript: <<
+  python%type_raw[python] setup.py build 
+<<
+
+InstallScript: <<
+  python%type_raw[python] setup.py install --root=%d
+<<
+
+InfoTest: <<
+  TestDepends: pytest-py%type_pkg[python], pretend-py%type_pkg[python]
```

Comment:
```
/sw/bin/python2.7 setup.py pytest || TESTFAIL=1
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'pytest'
```

Reply:
`%p/bin/pytest-%type_raw[python] || exit 2` works for TestScript and you can then get rid of the TESTFAIL and .pyc lines.

Reply:
That's odd. What is your version of `setuptools-tng-py27`? With `32.3.0-1` I got
```
/sw/bin/python2.7 setup.py pytest || TESTFAIL=1
running pytest
running egg_info
writing requirements to packaging.egg-info/requires.txt
writing packaging.egg-info/PKG-INFO
writing top-level names to packaging.egg-info/top_level.txt
writing dependency_links to packaging.egg-info/dependency_links.txt
reading manifest file 'packaging.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no previously-included files found matching '.travis.yml'
warning: no previously-included files found matching 'dev-requirements.txt'
no previously-included directories found matching 'docs/_build'
no previously-included directories found matching 'tasks'
writing manifest file 'packaging.egg-info/SOURCES.txt'
running build_ext
===================================================================== test session starts ======================================================================
platform darwin -- Python 2.7.14, pytest-3.4.0, py-1.5.2, pluggy-0.6.0
rootdir: /scratch.noindex/fink.build/packaging-py27-17.1-1/packaging-17.1, inifile:
collected 29047 items                                                                                                                                          

tests/test_markers.py .......................ss......................................................................................................... [  0%]
...
```
`%p/bin/pytest-%type_raw[python]` works the same here, but I seem to remember some packages where calling it directly failed to collect all tests...

Reply:
`i   setuptools-tng-py2  32.3.0-1`

Reply:
OK, I know what to add to `TestDepends` should I need it...
```
i pytest-runner-py27   3.0-1   Add setup.py test support
```
Leaving this package as is now.